api/components.yaml

```diff
@@ -0,0 +1,81 @@
+swagger: '2.0'
+definitions:
+  Error:
+    properties:
+      code:
+        format: int32
+        type: integer
+      message:
+        type: string
+    required:
+      - code
+      - message
+x-components:
+  parameters:
+    cursor:
+      description: An opaque identifier that points to the next item in the collection.
+      in: query
+      name: cursor
+      x-example: 01BX9NSMKVXXS5PSP2FATZM123
+      type: string
+    limit:
+      description: How many items to return at one time (max 100)
+      in: query
+      name: limit
+      format: int32
+      type: integer
+    offset:
+      description: The zero-ary offset index into the results
+      in: query
+      name: offset
+      default: 0
+      format: int32
+      type: integer
+    sort:
+      description: Sorting order
+      in: query
+      name: sort
+      default: 0
+      x-example: +name
+      type: string
+  responses:
+    400BadRequest:
+      content:
+        application/json:
+          schema:
+            $ref: '#/definitions/Error'
+      description: Bad Request
+    404NotFound:
+      content:
+        application/json:
+          schema:
+            $ref: '#/definitions/Error'
+      description: Not Found
+    429TooManyRequests:
+      content:
+        application/json:
+          schema:
+            $ref: '#/definitions/Error'
+      description: Too many requests
+    503ServiceUnavailable:
+      content:
+        application/json:
+          schema:
+            $ref: '#/definitions/Error'
+      description: Service Unavailable
+    default:
+      content:
+        application/json:
+          schema:
+            $ref: '#/definitions/Error'
+      description: Unexpected error
+  headers:
```

Comment:
@gunzip
While throttling it's not important at development time, it MUST be supported once they are exposed (Eg. via the api gateway). So it's fine for now, but put a note on the production environment ;)

Reply:
we do use throttling but there are no exposed headers with quotas (and with the current api gateway we have no chance to do that =)